cgpsa

```diff
@@ -2909,30 +3015,73 @@ sub readConfigFile
     {
       $conf->{sa_local_rules_dir} = $1;
     }
-    elsif ($line =~ /^absolute[-_]queue[-_]filenames[\s\=]+(\S+)$/) 
+    elsif ($line =~ /^absolute[-_]queue[-_]filenames[\s\=]+(\S+)$/)
     {
       $conf->{absolute_queue_filenames} = evaluateBoolean($1);
     }
     elsif ($line =~ /^prevent[-_]backscatter[\s\=]+(\S+)$/)
     {
       $conf->{prevent_backscatter} = evaluateBoolean($1);
     }
+    elsif ($line =~ /^x[-_]junk[-_]score[-_]header[\s\=]+(\S+)$/)
+    {
+      $conf->{x_junk_score_header} = evaluateBoolean($1);
+    }
+    elsif ($line =~ /^x[-_]junk[-_]score[-_]thresholds[\s\=]+(\S+)$/)
```

Comment:
The regex should be `(.+)` otherwise with `(\S+)` the config line would not be matched to the end.

Reply:
Another good catch.

Reply:
Can be resolved.